VERSION

```diff
@@ -0,0 +1 @@
+7.17.1-SNAPSHOT
```

Comment:
should be 7.18.0-SNAPSHOT as 7.17 was not tagged / not branched ?

Reply:
and/or create tag from last week commit and cherry-pick into that branch ?

Reply:
doesn't really matter as the value is meaningless, but if you WANT I can change it over. It'll be set to 7.18.1-SNAPSHOT in 2 weeks when we release 7.18.0

Reply:
https://github.com/eclipse/che-docs/pull/1463